catalyst/utils/dataset.py

```diff
@@ -53,9 +53,8 @@ def create_dataset(
     return dataset
 
 
-def split_dataset(
-    dataset: pd.DataFrame, **train_test_split_args
-) -> Tuple[DictDataset, DictDataset]:
+def split_dataset(dataset: pd.DataFrame,
```

Comment:
weird, but it's from flake8

Reply:
looks like flake8 version change, we need to fix this... issue :)

Reply:
done